.github/workflows/linux.yml

```diff
@@ -10,13 +10,16 @@ on:
 
 jobs:
   build:
+    strategy:
+      matrix:
+        go-version: [1.11, 1.12, 1.13, 1.14]
     runs-on: ubuntu-latest
 
     steps:
     - name: Set up Go
       uses: actions/setup-go@v1
       with:
-        go-version: 1.14
+        go-version: ${{ go-version }}
```

Comment:
@bradfitz Sorry, just noticed that this should have been `${{ matrix.go-version }}` and the action thus didn't run. Will send a follow-up PR.

Reply:
#23